client/src/app/pages/tasks/tasks-page.tsx

```diff
@@ -61,7 +61,11 @@ export const TasksPage: React.FC = () => {
 
   const tableControlState = useTableControlState({
     tableName: "tasks-table",
-    persistTo: { filter: "urlParams" },
+    persistTo: {
+      filter: "urlParams",
```

Comment:
urlParams instead of sessionStorage like the other 2?

Reply:
It's quite verbose and doesn't seem useful for copy-pasting manually i.e. to chat/mail.
Technically it would work. If you prefer this solution we can give a try.

Reply:
Really I just noticed the difference between application table and this table -- sessionStorage for filter in application table, and current in the filter here.  I don't really have a strong feeling either way.

Reply:
I would propose to test mixed approach and change later on if necessary.